index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <div class="images">
+    <img src="images/MyLogo.png" alt="my logo">
+    <img src="images/Union.png" alt="union">
```

Comment:
- [ ] Kindly wrap the hamburger menu in a `<nav>` tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.

Reply:
This is also an image i took from Figma website. So it doesn't have to work as dropdown. Please check the objectives.

Reply:
Hi, kindly try to understand that this is required due to semantic reasons as the previous reviewer has described. Also later on you will add a navigation menu that opens up when you click this button. We are required to check this too.

- The hamburger button is on the right and has a `nav` tag.
  * The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.